src/solvers/flattening/boolbv.cpp

```diff
@@ -249,7 +257,9 @@ bvt boolbvt::convert_bitvector(const exprt &expr)
     return convert_unary_minus(to_unary_expr(expr));
   else if(expr.id()==ID_unary_plus)
   {
-    assert(expr.operands().size()==1);
+    DATA_INVARIANT(
+      expr.operands().size() == 1,
+      "unary plus expressions should have one operand");
```

Comment:
@tautschnig Is likely going to suggest to add a `to_unary_plus_expr` and call it here...

Reply:
See PR #2931 ;)

Reply:
AFAIK that has been merged?